extension/src/experiments/data/index.ts

```diff
@@ -78,21 +83,23 @@ export class ExperimentsData extends BaseData<ExperimentsOutput> {
 
   private async collectGitLogByBranch(
     branch: string,
-    availableNbCommits: { [branch: string]: number }
+    availableNbCommits: { [branch: string]: number },
+    isCurrent?: boolean
   ) {
     const nbOfCommitsToShow = this.experiments.getNbOfCommitsToShow(branch)
+    const branchName = isCurrent ? gitPath.DOT_GIT_HEAD : branch
```

Comment:
[F] An explanatory variable would have been good here. The reason being when we come back to this we would have some idea as to why we made this decision. Example would be

- `detachedSafeBranchName` 
- `safeBranchName`

The information won't be lost as we can always come back to this PR (I find the PR via Gitlens file/line history) but you are saving your future self some work by fighting to make the code as expressive as possible 👍🏻.